packages/terra-demographics-banner/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Demographics Banner
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-demographics-banner.svg)](https://www.npmjs.org/package/terra-demographics-banner)
+[![Build Status](https://travis-ci.org/cerner/terra-ui.svg?branch=master)](https://travis-ci.org/cerner/terra-ui)
+
+{insert description}
```

Comment:
Can you paste the description from `docs/README.md` here. "The demographics component is used to display demographic information about a person in a condensed, easy to read format."